src/views/companyCheckStatus.html

```diff
@@ -0,0 +1,32 @@
+<main id="content" role="main">
+
+  {{> common/betaBanner }}
+
+  {{> common/backLink }}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <h1 id="page-heading" class="heading-large">{{ pageHeading }}</h1>
+
+      <form>
+        <div class="form-group">
+          <div id="confirm-result-message" class="panel panel-border-wide">
+            <p>
```

Comment:
We need to wrap the data below (company name, number, status) in SPANs and add IDs in here to enable testing.